core-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -26,13 +26,13 @@ appengine.run.no.gradle.state=App Engine Gradle configuration does not appear to
 
 appengine.sdk.invalid=A valid App Engine SDK was not found for module : {0}.
 
-clonefromgcp.projectid=Cloud Project
+clonefromgcp.projectid=Cloud Project\:
```

Comment:
I thought I'd backslashed it because it didn't show, but in my Mac it's always showing, so I'll just remove the backslash.